docs/competition/team-support/helpdesk.md

```diff
@@ -22,12 +22,10 @@ Helpdesk is the teams' primary source of support. Volunteers are expected to adv
  * Permanent marker
```

Comment:
(wrong line, but I can't comment far enough outside the diff):
There's a line here about a laptop for access to the IDE. Will all the volunteers on Helpdesk have access to the IDE? As far as I know, there's only a couple of people outside the Core Team who have accounts -- are we going to want a bunch more? (They _definitely_ need to be per-person as the accounts also provide access to the forum, for which we want to have per-person accounts)

Reply:
I'd assume we can log in to the IDE with the team's credentials, as I don't think volunteers can make edits to student code any way?

Reply:
Well yes, but they might as well bring their own laptops over in such cases.

Reply:
I don't feel strongly we need to change this.

Reply:
Ack, I imagine they probably bring their own laptops if it's a software issue anyway, so the laptop is probably just there to browse the docs.